cli.js

```diff
@@ -159,3 +204,7 @@ async function safeExecute(func) {
         await func()
     } catch {}
 }
+
+function isNotHiddenFile(src, dest) {
```

Comment:
why do we take `dest` as an argument here?

Reply:
I included it because I didn't know if I needed it and forgot to remove it. It's gone now